Xamarin.Forms.Platform.iOS/Extensions/ColorExtensions.cs

```diff
@@ -32,6 +32,15 @@ public static CGColor ToCGColor(this Color color)
 #endif
 		}
 
+		public static UIColor FromPatternImageFromBundle(string bgImage)
+		{
+			var image = UIImage.FromBundle(bgImage);
```

Comment:
Doesn't build on MacOS. Need to make an alias for the MacOS type. `NSImage`, I presume?

```  
/Users/vsts/agent/2.147.1/work/1/s/Xamarin.Forms.Platform.iOS/Extensions/ColorExtensions.cs(37,16): error CS0103: The name 'UIImage' does not exist in the current context 
```